### DISCUSSION TITLE
How to add a Span Link to a Span previously created by auto-instrumentation?

### DISCUSSION THREAD

**Discussion by philsttr** [open-telemetry/opentelemetry-java#6159] [Q&A]
I have a situation where I want to add a Span Link to a Span that was previously created via auto-instrumentation.  Since I can't modify the auto-instrumentation code to add the Span Link when it creates the Span, I would like to add the Span Link later _after_ it is created.  Is this possible somehow?

From what I can tell, Span Links can only be added via the `SpanBuilder` used to create the `Span`, and can't be added to the `Span` later, but I might be missing something.

**Comment by trask** [comment]
related to https://github.com/open-telemetry/opentelemetry-specification/pull/3678
